lib/console.ex

```diff
@@ -41,8 +40,8 @@ defmodule Console do
     IO.write message
   end
 
-  def play_again do
-    String.trim(IO.gets %Message{}.play_again) |> String.downcase()
+  def prompt_new_game do
```

Comment:
I like this, it's an adjacent label move but it definitely makes the code easier to understand.